src/Microsoft.TestPlatform.Utilities/XmlUtilities.cs

```diff
@@ -63,7 +63,7 @@ internal static bool IsValidNodeXmlValue(string xmlNodeValue, Func<string, bool>
         var secureInnerXml = SecurityElement.Escape(innerXml);
 #else
         // fixing manually as we currently target to netcore 1.1 and we don't have default implementation for Escape functionality
-        var secureInnerXml = innerXml.IsNullOrEmpty() ? innerXml : innerXml.Replace("&", "&amp;").Replace("<", "&lt;").Replace(">", "&gt;").Replace("\"", "&quot;").Replace("'", "&apos;");
+        var secureInnerXml = (innerXml ?? string.Empty).Replace("&", "&amp;").Replace("<", "&lt;").Replace(">", "&gt;").Replace("\"", "&quot;").Replace("'", "&apos;");
```

Comment:
You have changed the logic here. That's not a big deal but you are forcing unnecessary operations